pkg/gateway/model_build_targets.go

```diff
@@ -107,7 +109,24 @@ func (t *latticeTargetsModelBuildTask) buildLatticeTargets(ctx context.Context)
 		errmsg := fmt.Sprintf("Build Targets failed because K8S service %v does not exist", namespacedName)
 		return errors.New(errmsg)
 	}
+
+	portAnnotations := int64(0)
```

Comment:
NIT,  use a constant (e.g. #define NO_PORT_SELECTOR,  and portAnnotations := NO_PORT_SELECTOR) to make it more readable

Reply:
Done